package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.5.4",
   "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
   "main": "snabbdom.js",
-  "typings": "type-definitions/snabbdom.d.ts",
+  "typings": "snabbdom.d.ts",
```

Comment:
Yeah, this allows the TypeScript compiler to find the associated typings automatically via NPM. So the user doesn't have to do anything else other than `npm install --save snabbdom`.